### DISCUSSION TITLE
Valent in combination with audio, bluetooth or otherwise...

### DISCUSSION THREAD

**Discussion by johnea** [andyholmes/valent#307] [Help]
Hello, Thanks for the great work!

This is most likely an uninformed question, but:
 Can valent be used in combination with bluetooth (or other) audio to fully operate a cell phone remotely? Including both making and receiving voice calls?

This would be in combination with KDE Connect running on an android phone.

Maybe there are other means of audio forwarding, such as over USB?

The gist of the question is whether valent can be used in combination with say a headset jumper cable, or bluetooth audio.

It's a much nicer solution if all the control and audio are in the same application together, but is this really necessary?

Setting up one application to tunnel audio, and another (valent) to remote the phone display, remote screen touch, and receive notifications, seems like a totally valid solution.

So, just wondering if this level of complete remote operation could be reached using valent in combination with other s/w, and what you feel is the shortest path, at present, to achieving this functionality.

Thanks again for the great project!

John EA
2022-10-21

**Comment by andyholmes** [comment]
> Can valent be used in combination with bluetooth (or other) audio to fully operate a cell phone remotely? Including both making and receiving voice calls?

It can not, no.

> This would be in combination with KDE Connect running on an android phone.
> Maybe there are other means of audio forwarding, such as over USB?

KDE Connect can do neither of these things, so Valent can't either unfortunately.

> So, just wondering if this level of complete remote operation could be reached using valent in combination with other s/w, and what you feel is the shortest path, at present, to achieving this functionality.

It could be done, but someone would have to write the code. I'm not sure what the shortest path to do all that is to be honest, but I doubt there's one that doesn't involve a lot of work on your part  :slightly_smiling_face:
